### PR TITLE
Changed default font family

### PR DESCRIPTION
**Pull request by NaughtyChas**
I've changed the font of our website to `Torus` to make it looks more osu!-ish:

|Navbar|Footer|
|:-:|:-:|
|Navbar|image|

### CODE REVIEW

**Comment by CloneWith** [comment]
The font looks good. Have you considered defining a font for CJK characters (like *homoss*)?

**Comment by offfc** [comment]
> The font looks good. Have you considered defining a font for CJK characters (like _homoss_)?

Not yet. Maybe in the next commit.

**Comment by NaughtyChas** [comment]
> The font looks good. Have you considered defining a font for CJK characters (like _homoss_)?

Not yet, maybe in the next commit.

File: src/css/custom.css [hunk 1]
```diff
@@ -24,10 +24,22 @@
...
+    font-family: "HarmonyOS Sans";
+    src: url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Black.ttf") format("truetype"), /* Add other weights/styles as needed */
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Bold.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Light.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Medium.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Regular.ttf") format("truetype"),
+    url("/static/fonts/HarmonyOS Sans/HarmonyOS_Sans_Thin.ttf") format("truetype");
+    unicode-range: U+4E00-9FFF, U+3400-4DBF, U+20000-2A6DF, U+2A700-2B73F, U+2B740-2B81F, U+2B820-2CEAF;
+  }
+
```
Comment (NaughtyChas) [inline src/css/custom.css:40]: Use `HarmonyOS Sans` for CJK characters display.

Comment (CloneWith) [reply src/css/custom.css:40]: Can consider adding 3000-303F to this range, which consists of some common CJK symbols and punctuations.

table

Comment (NaughtyChas) [reply src/css/custom.css]: in commit [65d95f0](https://github.com/offfc/offfc.github.io/pull/5/commits/65d95f092de8f20cb7f18ab5c9d47e15d754d17b).

Comment (CloneWith) [reply src/css/custom.css]: After a time of research, I tried to make the family definition at the root level of the stylesheet, and somehow it worked:

mixed fonts

How strange.

Comment (NaughtyChas) [reply src/css/custom.css]: Great.

Push that commit onto this branch and we will merge this PR.

**Comment by CloneWith** [comment]
Don't know why building the site will cause some strange behaviours, like the font won't work properly... Gotta smoke.

**Review by CloneWith — APPROVED** [review]

**Pull request merged**